frontend/packages/console-shared/src/components/dashboard/status-card/OperatorStatusBody.tsx

```diff
@@ -27,12 +29,16 @@ export const OperatorsSection: React.FC<OperatorsSectionProps> = ({
       <div className="co-status-popup__row">
         <div>
           <span className="co-status-popup__text--bold">{title}</span>
-          <span className="text-secondary">{` (${operatorStatuses.length} installed)`}</span>
+          <span className="text-secondary">
+            {t('console-shared~ ({{operatorStatusLength}} installed)', {
+              operatorStatusLength: operatorStatuses.length,
+            })}
+          </span>
```

Comment:
What does the final string look like? This to me reads like a single line of text and therefore should be translated using `Trans` component with two variables.

Reply:
This is how it looks
![wl](https://user-images.githubusercontent.com/22490998/107759831-e01ff700-6d4e-11eb-973f-d9052fab00db.png)
![lo](https://user-images.githubusercontent.com/22490998/107759838-e1512400-6d4e-11eb-9646-fe843aa42bbe.png)